prov/rxm/src/rxm_conn.c

```diff
@@ -561,8 +561,12 @@ rxm_process_reject(struct rxm_conn *conn, struct fi_eq_err_entry *entry)
 		break;
 	case RXM_CM_CONNECTING:
 		rxm_close_conn(conn);
-		if (reason != RXM_REJECT_EALREADY)
+		if (reason != RXM_REJECT_EALREADY) {
 			rxm_free_conn(conn);
+		}
+		else {
+			FI_INFO(&rxm_prov, FI_LOG_EP_CTRL, "rejected, already connected\n");
+		}
```

Comment:
Please fix formatting.  In this case, braces are not needed.  If they were, they should be on the same line as the else.